tests/jobs/test_triggerer_job.py

```diff
@@ -305,22 +303,14 @@ def test_invalid_trigger(session):
     session.commit()
 
     # Create the test DAG and task
-    with DAG(
-        dag_id='test_invalid_trigger',
-        start_date=timezone.datetime(2016, 1, 1),
-        schedule_interval='@once',
-        max_active_runs=1,
-    ):
-        task1 = DummyOperator(task_id='dummy1')
+    with dag_maker(dag_id='test_invalid_trigger'):
+        DummyOperator(task_id='dummy1')
 
+    dr = dag_maker.create_dagrun()
```

Comment:
Do we need to pass `session` in?

Reply:
Done in 2a7f8a411 (to the dag_maker call)